### DISCUSSION TITLE
Please add swedish BankId as a authProvider

### DISCUSSION THREAD

**Discussion by jonaed1230** [nextauthjs/next-auth#4050] [Ideas]
### Description 📓

Most Swedish users love to use BankId as their authentication system. Please implement it with the next auth. Currently, I'm having pain in setup the BankId auth system with my nodeJS app Because it's not well documented.
BankId link: https://www.bankid.com/en
BankId Docs Link: https://www.bankid.com/en/utvecklare/guider

### How to reproduce ☕️

.

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
From what I could gather from their documentation, they are not using OAuth or OIDC, so at best, we could only support it with a Custom provider, probably Credentials or Email. This is unlikely to make it into the list of officially supported providers.
